Dockerfile

```diff
@@ -87,17 +105,15 @@ RUN julia --eval 'using Pkg; pkg"precompile"'
 
 USER root
 # Example Data
-RUN mkdir /data
-RUN mkdir /data/Diva-Workshops-data
+RUN mkdir -p /data/Diva-Workshops-data && \
+    chown jovyan:jovyan /data -R
+USER jovyan
+
 RUN curl https://dox.ulg.ac.be/index.php/s/Px6r7MPlpXAePB2/download | tar -C /data/Diva-Workshops-data -zxf -
 RUN ln -s /opt/julia-* /opt/julia
```

Comment:
@Marie59 what is the intension here? We are installing Julia with conda to make the notebook work, but here you are downloading Julia and use a different version that we installed previously.

Julia is available in version 1.8 and 1.9 https://anaconda.org/conda-forge/julia

Reply:
Done !